src/primitives/transaction.h

```diff
@@ -28,6 +28,9 @@ static const int SERIALIZE_TRANSACTION_NO_WITNESS = 0x40000000;
 // Globals to avoid circular dependencies.
 extern bool g_con_elementsmode;
 
+// SEQUENTIA:
+extern bool g_con_sequentiamode;
```

Comment:
There are two other header files where `g_con_elementsmode` is defined, so this might need to be added there as well. It would be better to only rely on one, though, and understand why elements has three.